ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,1413 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    endpoint = aiplatform.Endpoint(endpoint_name)
</pre><p>Again, you could just have assigned to <code>endpoint</code>  when doing deployment:</p><p> </p><pre class="ql-syntax" spellcheck="false">endpoint = model.deploy(deployed_model_display_name=deployed_model_display_name, sync=True)
</pre><p> </p><p>Are you showing it this way as you want to show how to list endpoints?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>See previous comment w/r/t listing and getting resources.</p>